docker-compose.yml

```diff
@@ -252,15 +252,15 @@ services:
     command: subscriptions --auto-offset-reset=latest --consumer-group=snuba-transactions-subscriptions-consumers --topic=events --result-topic=transactions-subscription-results --dataset=transactions --commit-log-topic=snuba-commit-log --commit-log-group=transactions_group --delay-seconds=60 --schedule-ttl=60
   snuba-cleanup:
     <<: *snuba_defaults
-    image: snuba-cleanup-onpremise-local
+    image: snuba-cleanup-self-hosted-local
```

Comment:
This might leave some extraneous images hanging around people's environments but I don't think that's any great problem?

Reply:
This should be fine. They can be removed with `--remove-orphans`